src/index.js

```diff
@@ -11,8 +13,9 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
 app.use(cors());
-
-app.use('/api', routes);
+app.use('/auth', auth);
+app.use(authenticateUser);
+app.use('/api', api);
```

Comment:
```suggestion
app.use('/api', authenticateUser, api);
```

This allows all calls to the `/api` route to be authenticated